### DISCUSSION TITLE
[FEATURE]: Allow public access to discounted prices on active listings

### DISCUSSION THREAD

**Discussion by wessells** [etsy/open-api#716] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

I'd like to analyse groups of items based on multiple variables, one of which being their prices, but there is no way to accurately portray prices of items without being able to access the current discount on something.

Discounts are currently accessible only in the receipts of items, but that is behind an auth2 wall and also not on active listings.

### Describe the solution you'd like

getListing could get an extra field of information: "Discount" with a percentage (or dollar value) off.

Maybe a listing could keep a 'price.amount' and a 'price.discounted' or something like that.

**Comment by etsyachristensen** [comment]
We are converting this to a discussion in the Idea category to continue tracking the ticket.
